{"name": "test_nobeam", "status": "passed", "description": " test having no beam throws an exception", "attachments": [{"name": "stderr", "source": "9255abd6-64d3-4d68-9baa-fd87de03214a-attachment.txt", "type": "text/plain"}], "start": 1758034596866, "stop": 1758034596867, "uuid": "d2dee51b-bd83-433b-9028-493a74d7c54a", "historyId": "90ca090201cb70956efeb3be9708d0c0", "testCaseId": "90ca090201cb70956efeb3be9708d0c0", "fullName": "casatools.test_tool_image.ia_commonbeam_test#test_nobeam", "labels": [{"name": "parentSuite", "value": "casatools"}, {"name": "suite", "value": "test_tool_image"}, {"name": "subSuite", "value": "ia_commonbeam_test"}, {"name": "host", "value": "casa-rockylinux8-amd64-perf07"}, {"name": "thread", "value": "593221-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "casatools.test_tool_image"}], "titlePath": ["casatools", "test_tool_image.py", "ia_commonbeam_test"]}